File: app/src/main/java/forpdateam/ru/forpda/presentation/history/HistoryView.java
package forpdateam.ru.forpda.presentation.history;

import java.util.List;

import forpdateam.ru.forpda.common.mvp.IBaseView;
import forpdateam.ru.forpda.entity.app.history.HistoryItem;

/**
 * Created by radiationx on 01.01.18.
 */

public interface HistoryView extends IBaseView {
    void showHistory(List<HistoryItem> items);
}

File: app/src/main/java/forpdateam/ru/forpda/presentation/history/HistoryView.kt
package forpdateam.ru.forpda.presentation.history

import forpdateam.ru.forpda.common.mvp.IBaseView
import forpdateam.ru.forpda.entity.app.history.HistoryItem
import moxy.viewstate.strategy.AddToEndSingleStrategy
import moxy.viewstate.strategy.SkipStrategy
import moxy.viewstate.strategy.StateStrategyType

/**
 * Created by radiationx on 01.01.18.
 */

@StateStrategyType(AddToEndSingleStrategy::class)
interface HistoryView : IBaseView {
    fun showHistory(items: List<HistoryItem>)

    @StateStrategyType(SkipStrategy::class)
    fun showItemDialogMenu(item: HistoryItem)
}
